src/pluto/helpers.py

```diff
@@ -35,6 +35,19 @@ def get_data(branch) -> Dict[str, pd.DataFrame]:
         kwargs={"converters": {"outlier": json.loads}} | kwargs,
     )
 
+    rv = rv | get_corrections(client, branch)
```

Comment:
What is this syntax lol

Reply:
As far as I know, it's the easiest way to merge two dictionaries in python.

Reply:
https://stackoverflow.com/questions/38987/how-do-i-merge-two-dictionaries-in-a-single-expression

Reply:
Awesome thanks, I couldn't find that by googling. Are you against passing the function `rv` and modifying it in place? That seems more intuitive to me than starting a second `rv`. Not that it matters just wanted to pick your brain

Reply:
Can you even do that? I think python passes things by value, not reference. So the rv that would get passed to the function is a copy, and changes you make to it won't affect the one back in the parent function. I don't really know tbh, but i think it requires a deep understanding of the language to know if that will work. Either way, i think its maybe better that this separate function is self-contained and can't mutate an unrelated dictionary other things rely on

Reply:
Great question! It turns out you can, python points to the same place in memory for a list or dictionary even as it's passed between stack frames. The technical term for this is "mutability" https://www.dataquest.io/blog/tutorial-functions-modify-lists-dictionaries-python/ 
To me there is one `rv` that is created at the top of get_data and then everything should be added to this same rv without any intermediaries. But I agree that your design is less fragile